samples/wcfapp/WcfServiceConsoleApp/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM microsoft/dotnet-framework:4.6.2
```

Comment:
What would it take to have a multi-stage Dockerfile that builds the project with one stage and produces the "runtime" image in another?  Can you make use of microsoft/dotnet-framework:4.7.2-sdk to build?  We should be supporting containerized builds.

Reply:
@MichaelSimons In this sample, after WCF services are running in the container, users should be able to run WCF client against them. If the projects are build inside container, users still need to build the project locally to have WcfClient.exe. Am I understanding it correctly?